### ISSUE TITLE
Batch edit (megaticket)

### ISSUE THREAD

**Issue by julyytran** [ctoec/ecis-experimental#941]
As a provider who has missing enrollment data for many children, I can see and update what information is missing more easily so that I can spend less time tracking data.

### **Mockups & Specs**
- [Mockups here](https://skylight.invisionapp.com/overview/REVIEWBatchEdit-ck8kdn9c5000c0136gpjvksq1/screens?v=k7wjDK1wn75C4uQO811F0w%3D%3D&linkshare=urlcopied) 
- [Specs here](https://skylight.invisionapp.com/console/HANDOFF_BatchEdit-Specs-ck8yt41bw00lt010ag6mt32gw/ck8yt41vo00lu010am5yf2bba/play#project_console)

### **Batch Edit View**
- [x] Sidenav List component #945 
- [x] Modular enrollment section fields #946
- [x] Batch edit #947 (blocked by #945, #946)

### **Pending report alert**
- [x] #1210 Pending report with missing info has alert that takes me to batch edit

**Comment by julyytran** [comment]
Notes from backlog refinement

Turn into a megaticket checklist and break up into multiple stories:
- roster update
- components for new batch edit
- container for new components

Megaticket has blocked labels for what blocks it to capture order of operations

AC updates
- income determination/redetermination
- if child not missing birt cert or income, text for # of fields missing
- needed info alert copy update
- order by last name, no age group order
- only from program view, not site view

**Comment by julyytran** [comment]
@jlhogan I updated the AC and split out the roster piece into a separate story. Could you split it up further and add blocking labels?

**Comment by jlhogan** [comment]
@julyytran done!  lmk if any of the changes I made to AC are unclear
